### ISSUE TITLE
can't delete packaged folders

### ISSUE THREAD

**Issue by PierBover** [electron-userland/electron-packager#401]
I've built to `~/Desktop` but I can't delete the packaged folder.

OSX asks for my password but then nothing happens after deleting. All the packaged folders remain there.

I've had to manually do `rm -rf` to remove the folders.

I'm on 10.11.5

Is this me, or anyone else experiencing this?

> Which version of electron-packager are you using?

7.0.4

> What version of [Electron](http://electron.atom.io) are you building with?
1.2.2

> What is the host platform are you running electron-packager on?
all

> What target platform(s)/architecture(s) are you building for?
all

> Is there a stack trace in the error message you're seeing?
no

**Comment by jprichardson** [comment]
Yes. Duplicate: https://github.com/electron-userland/electron-packager/issues/375
